doc/src/SUMMARY.md

```diff
@@ -1,6 +1,6 @@
 # Summary
 
-[Overview](overview.md)
+- [Overview](overview.md)
```

Comment:
what's the reason for this bullet point? There is only one overview

Reply:
Being consistent with the `Contributing` section? Can remove the bullet from both